Geometry/TrackerGeometryBuilder/test/python/testPixelTopologyMapTest_cfg.py

```diff
@@ -30,8 +30,8 @@
 process.load("Configuration.StandardSequences.Services_cff")
 
 if 'phase2' in options.globalTag:
-    process.load("Configuration.Geometry.GeometryExtended2026D92_cff")
-    process.load("Configuration.Geometry.GeometryExtended2026D92Reco_cff")
+    process.load("Configuration.Geometry.GeometryExtended2026Default_cff")
+    process.load("Configuration.Geometry.GeometryExtended2026DefaultReco_cff")
```

Comment:
unfortunately when running `scram b runtests_GeometryTrackerGeometryBuilderTestDriver` this leads to:

```console
Begin processing the 1st record. Run 1, Event 1, LumiSection 1 on stream 0 at 21-Aug-2024 12:58:36.284 CEST
===== Testing Phase-2 Pixel Tracker geometry =====


A fatal system signal has occurred: segmentation violation
The following is the call stack containing the origin of the signal.

Wed Aug 21 12:58:39 CEST 2024
Thread 2 (Thread 0x7f08803aa640 (LWP 4130827) "cmsRun"):
#0  0x00007f08a62d8a3f in wait4 () from /lib64/libc.so.6
#1  0x00007f08a280f3f7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/el9_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/pluginFWCoreServicesPlugins.so
#2  0x00007f08a281222a in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/el9_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/pluginFWCoreServicesPlugins.so
#3  0x00007f08a66d84d3 in std::execute_native_thread_routine (__p=0x7f0882f69e30) at ../../../../../libstdc++-v3/src/c++11/thread.cc:82
#4  0x00007f08a6289c02 in start_thread () from /lib64/libc.so.6
#5  0x00007f08a630ec40 in clone3 () from /lib64/libc.so.6
Thread 1 (Thread 0x7f08a76d3680 (LWP 4130817) "cmsRun"):
#0  0x00007f08a63019ff in poll () from /lib64/libc.so.6
#1  0x00007f08a2812447 in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/el9_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/pluginFWCoreServicesPlugins.so
#2  0x00007f08a2812644 in sig_dostack_then_abort () from /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/el9_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/pluginFWCoreServicesPlugins.so
#3  <signal handler called>
#4  0x00007f087a88f7fc in PixelTopologyMap::buildTopologyMaps() () from /tmp/musich/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/libGeometryTrackerGeometryBuilder.so
#5  0x00007f087403788d in PixelTopologyMapTest::analyze(edm::StreamID, edm::Event const&, edm::EventSetup const&) const () from /tmp/musich/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/pluginPixelTopologyMapTest.so
#6  0x00007f08a8642dca in edm::global::EDAnalyzerBase::doEvent(edm::EventTransitionInfo const&, edm::ActivityRegistry*, edm::ModuleCallingContext const*) () from /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/el9_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/libFWCoreFramework.so
#7  0x00007f08a863cfbc in edm::WorkerT<edm::global::EDAnalyzerBase>::implDo(edm::EventTransitionInfo const&, edm::ModuleCallingContext const*) () from /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/el9_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/libFWCoreFramework.so
#8  0x00007f08a85c25f9 in std::__exception_ptr::exception_ptr edm::Worker::runModuleAfterAsyncPrefetch<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(std::__exception_ptr::exception_ptr, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*) () from /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/el9_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/libFWCoreFramework.so
#9  0x00007f08a85c2ae4 in edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute() () from /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/el9_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/libFWCoreFramework.so
#10 0x00007f08a8344248 in tbb::detail::d1::function_task<edm::WaitingTaskList::announce()::{lambda()#1}>::execute(tbb::detail::d1::execution_data&) () from /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/el9_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/libFWCoreConcurrency.so
#11 0x00007f08a7ea33a1 in tbb::detail::r1::task_dispatcher::local_wait_for_all<false, tbb::detail::r1::external_waiter> (waiter=..., t=<optimized out>, this=0x7f08a49c3e00) at /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el9_amd64_gcc12/external/tbb/v2021.9.0-f55817ca40e6490787280b5021409fd9/tbb-v2021.9.0/src/tbb/task_dispatcher.h:322
#12 tbb::detail::r1::task_dispatcher::local_wait_for_all<tbb::detail::r1::external_waiter> (waiter=..., t=<optimized out>, this=0x7f08a49c3e00) at /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el9_amd64_gcc12/external/tbb/v2021.9.0-f55817ca40e6490787280b5021409fd9/tbb-v2021.9.0/src/tbb/task_dispatcher.h:458
#13 tbb::detail::r1::task_dispatcher::execute_and_wait (t=<optimized out>, wait_ctx=..., w_ctx=...) at /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el9_amd64_gcc12/external/tbb/v2021.9.0-f55817ca40e6490787280b5021409fd9/tbb-v2021.9.0/src/tbb/task_dispatcher.cpp:168
#14 0x00007f08a854420b in edm::FinalWaitingTask::wait() () from /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/el9_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/libFWCoreFramework.so
#15 0x00007f08a8550f3c in edm::EventProcessor::processRuns() () from /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/el9_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/libFWCoreFramework.so
#16 0x00007f08a8551391 in edm::EventProcessor::runToCompletion() () from /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/el9_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-08-20-2300/lib/el9_amd64_gcc12/libFWCoreFramework.so
#17 0x0000000000408431 in tbb::detail::d1::task_arena_function<main::{lambda()#1}::operator()() const::{lambda()#1}, void>::operator()() const ()
#18 0x00007f08a7e8f96d in tbb::detail::r1::task_arena_impl::execute (ta=..., d=...) at /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el9_amd64_gcc12/external/tbb/v2021.9.0-f55817ca40e6490787280b5021409fd9/tbb-v2021.9.0/src/tbb/arena.cpp:688
#19 0x000000000040a13c in main::{lambda()#1}::operator()() const ()
#20 0x0000000000405100 in main ()

Current Modules:

Module: PixelTopologyMapTest:myanalysis (crashed)
```

Reply:
For the record, the issue appears when processing detid `303042564` (it worked correctly for detid `303042565`) when trying to access the layer from the topology:

https://github.com/cms-sw/cmssw/blob/d34e3af53bf3f1fe6e8041c0980f8b3a6d2f064e/Geometry/TrackerGeometryBuilder/src/PixelTopologyMap.cc#L26-L31

Reply:
Fixed by https://github.com/cms-sw/cmssw/pull/45764/commits/8fe87ed9d6f1abfd296099a986c3f6d1ef2e1f69